methods/coseismic/Coseismic_Requirement_Validation-Ridgecrest_asc.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        command = 'ariaDownload.py --bbox ' + download_region + ' --start ' + download_start_date + ' --end ' + download_end_date 
</pre><p>Here we would add and else and then pull the fully staged data from S3. Would run through the notebook once till the end and then zip up everyhting and that is what we stage. We might need to check the png or some figure that might get written as i am not sure how the notebook reacts when re-running.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I agree that it would be good to zip up the whole MintPy directory at the end. I don't think there is any need to stage the ARIA-tools processed stack files.</p>